src/istep/help/P10/istep_list.htxt

```diff
@@ -0,0 +1,210 @@
+isteps for IP P10 Systems
```

Comment:
Paths need to be lower case (p10 not P10), and should be organized so everything for p10 can go under that one directory.  I would suggest `src/istep/p10/help/istep_list.htxt` and then future generations can follow the same layout.